lib/DeonApi.ts

```diff
@@ -36,6 +36,7 @@ interface CslApi {
 
 interface InfoApi {
   get(): Promise<D.NodeInfoOutput>;
+  requestAgent(s: string): Promise<D.AgentValue>;
```

Comment:
What is `s`? Could we come up with a more descriptive name?

Reply:
Yes we can!